amplify/backend/function/fetchMotionState/src/utils.js

```diff
@@ -218,6 +218,7 @@ const updateMotionMessagesInDB = async (
 
   motionMessages.forEach((message) => {
     updatedMessages.push({
+      initiatorAddress: constants.AddressZero,
```

Comment:
Since SystemMessages do not have an initiator when saved to the db during fetchMotionState, AddressZero is used as a placeholder for initiatorAddress.